l10n_it_vat_registries_cash_basis/models/account.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Lara Baggio - Link IT srl
+# (<http://www.linkgroup.it/>)
+# Copyright 2014-2017 Lorenzo Battistini - Agile Business Group
+# (<http://www.agilebg.com>)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    def _set_journal_domain(self, domain):
+        # aggiungo i movimenti dei giroconti dei pagamenti del journal
+        # dedicato al cash_basis
+        new_domain = []
+        journal_id = self.env.context['vat_registry_journal_ids']
+        cb_journal_id = self.env.user.company_id.tax_cash_basis_journal_id
+        journal_id.append(cb_journal_id.id)
+
+        for condition in domain:
+            if condition[0] == 'move_id.journal_id':
+                new_domain.append(('move_id.journal_id', 'in', journal_id))
+            else:
+                new_domain.append(condition)
+
+        return new_domain
+
+    def get_balance_domain(self, state_list, type_list):
+        domain = super(AccountTax, self).get_balance_domain(
+            state_list, type_list)
+
+        if not self.env.context.get('vat_registry_journal_ids'):
+            return domain
+
+        return self._set_journal_domain(domain)
+
+    def get_base_balance_domain(self, state_list, type_list):
+        domain = super(AccountTax, self).get_base_balance_domain(
+            state_list, type_list)
+
+        if not self.env.context.get('vat_registry_journal_ids'):
+            return domain
+
+        return self._set_journal_domain(domain)
+
+    def _get_move_invoice_from_reconcilie(self, move_id):
+        move_line_id = move_id.tax_cash_basis_rec_id.credit_move_id
+        if move_line_id.invoice_id:
+            return move_line_id.move_id
+        move_line_id = move_id.tax_cash_basis_rec_id.debit_move_id
+        if move_line_id.invoice_id:
+            return move_line_id.move_id
+
+        return None
+
+    def compute_balance(self, tax_or_base='tax', move_type=None):
+        self.ensure_one()
```

Comment:
In questo caso nn capisco perché dovrei chiamare il super. 
Ha senso chiamare il super se si sta estendendo una funzione, ma in tutti i casi che hai indicato sono delle sostituzioni.
Chiamandolo farei un operazione inutile e il risultato sarebbe scartato. 
C'è qualcosa che mi sfugge?